src/helpers/tests/geoChart/geoChart.spec.ts

```diff
@@ -102,6 +103,47 @@ describe("getGeoData", () => {
         });
     });
 
+    describe("isClusteringAllowed", () => {
+        it.each([
+            [
+                false,
+                "size measure",
+                {
+                    location: { index: 0, name: "location" },
+                    size: { index: 0, name: "size" },
+                },
+            ],
+            [
+                false,
+                "color measure",
+                {
+                    location: { index: 0, name: "location" },
+                    color: { index: 1, name: "color" },
+                },
+            ],
+            [
+                false,
+                "segment attribute",
+                {
+                    location: { index: 0, name: "location" },
+                    segment: { index: 1, name: "segment" },
+                },
+            ],
+            [
+                true,
+                "no others",
+                {
+                    location: { index: 0, name: "location" },
+                },
+            ],
+        ])(
+            "should return %s when %s exists",
```

Comment:
it could be `should return %s when %s is in bucket`

Reply:
updated